appveyor.yml

```diff
@@ -1,153 +1,122 @@
-os: Windows Server 2012
```

Comment:
Did the line endings change in this file? Why is everything marked as deleted and then re-added?

Reply:
Yeah, my ide is using CRLF in default i had to change it fo LF.

Reply:
@ahmadshabib are you using windows? if so you should turn autocrlf on as described in http://checkstyle.sourceforge.net/beginning_development.html#Before_development .

Reply:
@romani ping because of other conversations going on about `autocrlf`.